### PR TITLE
Add links to flowforge.com for easier navigation to our website.

### PR DESCRIPTION
**Pull request by iskerrett**
## Description

Added links to flowforge.com to make it easier for people to navigate to our website.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # FlowForge
 
+https://flowforge.com/
```
Comment (Pezmc) [inline README.md:3]: This floating at the top of the docs on GitHub looks out of place.

GitHub has an inbuilt repository about section, that's likely a better place for this:

Screenshot 2023-06-27 at 15 58 00

Someone with the correct GitHub permissions can add the link.

Comment (knolleary) [reply README.md:3]: There are a few places we can add the link... but having it in the readme is important (aside from exact placement etc) as that will also end up on the npm page for our package - https://www.npmjs.com/package/@flowforge/flowforge

Comment (Pezmc) [reply README.md:3]: On NPM we could also update the website field to actually point to the website. I believe that's pulled from package.json's homepage field.

Comment (hardillb) [reply README.md:3]: I've updated the github website field (and added `node-red` and `low-code` keywords)

Comment (knolleary) [reply README.md]: I have updated the `homepage` field and added the company logo as the link to the homepage in the readme.

**Review by knolleary — APPROVED** [review]

**Pull request merged**